zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -602,6 +602,10 @@ public boolean isWindowsPath(String path){
     return path.matches("^[A-Za-z]:\\\\.*");
   }
 
+  public boolean isHdfsPath(String path){
+      return path.matches("^hdfs://.*");
+  }
+
```

Comment:
Theoretically we can use FileSystemStorage with HDFS compatible file systems. For example, I'm using `dsefs` (from DataStax) very often as replacement for HDFS.

I maybe instead would go with something like this:

```scala
return new org.apache.hadoop.fs.Path(path).toUri.getScheme != null // or check for isBlank?
```

overall, I would maybe switch to more high-level API, like, Java's `Path`, Hadoop Path, etc. instead of doing "manual checks"

@zjffdu WDYT?

Reply:
I know the check is very basic. You are right, we should improve the check. I am open to suggestions.

Reply:
you can start with proposed code, and maybe exclude http schema, etc.

Reply:
Why exclude something, for example http?

I think we should avoid a dependency to hadoop in this code part.

Reply:
ok, then we can just try to instantiate URI, although we maybe need to wrap it in try/catch...

another possibility is to modify regex to match scheme as `scheme://.+`, with scheme regex as defined per RFC 3986: `scheme      = ALPHA *( ALPHA / DIGIT / "+" / "-" / "." )`

Reply:
Thanks for your feedback. I updated the code to the URI approach. I know the try/catch is not nice, but I think this solution is more future-proof.

Reply:
Thank you!